generated/nirfmxgsm/nirfmxgsm.proto

```diff
@@ -31,35 +932,1061 @@ message CloseResponse {
+  double switching_carrier_power = 2;
+  repeated double lower_relative_power = 3;
+  repeated double upper_relative_power = 4;
+  repeated double lower_absolute_power = 5;
+  repeated double upper_absolute_power = 6;
+  int32 actual_array_size = 7;
+}
+
+message PVTCfgAveragingRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof averaging_enabled_enum {
+    PvtAveragingEnabled averaging_enabled = 3;
+    int32 averaging_enabled_raw = 4;
+  }
+  int32 averaging_count = 5;
+  oneof averaging_type_enum {
+    PvtAveragingType averaging_type = 6;
+    int32 averaging_type_raw = 7;
+  }
+}
+
+message PVTCfgAveragingResponse {
+  int32 status = 1;
+}
+
+message PVTFetchMeasurementStatusRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message PVTFetchMeasurementStatusResponse {
+  int32 status = 1;
+  PvtMeasurementStatus measurement_status = 2;
+  int32 measurement_status_raw = 3;
+}
+
+message PVTFetchPowerTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message PVTFetchPowerTraceResponse {
+  int32 status = 1;
+  double x0 = 2;
+  double dx = 3;
+  repeated float upper_mask = 4;
+  repeated float signal_power = 5;
+  repeated float lower_mask = 6;
+  int32 actual_array_size = 7;
+}
+
+message PVTFetchSlotMeasurementRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message PVTFetchSlotMeasurementResponse {
+  int32 status = 1;
+  double slot_average_power = 2;
+  double slot_burst_width = 3;
+  PvtSlotMeasurementStatus slot_measurement_status = 4;
+  int32 slot_measurement_status_raw = 5;
+  double slot_maximum_power = 6;
+  double slot_minimum_power = 7;
+  double slot_burst_threshold = 8;
+}
+
+message PVTFetchSlotMeasurementArrayRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message PVTFetchSlotMeasurementArrayResponse {
+  int32 status = 1;
+  repeated double slot_average_power = 2;
+  repeated double slot_burst_width = 3;
+  repeated PvtSlotMeasurementStatus slot_measurement_status = 4;
+  repeated int32 slot_measurement_status_raw = 5;
+  repeated double slot_maximum_power = 6;
+  repeated double slot_minimum_power = 7;
+  repeated double slot_burst_threshold = 8;
+  int32 actual_array_size = 9;
+}
+
+message ResetAttributeRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  NiRFmxGSMAttribute attribute_id = 3;
+}
+
+message ResetAttributeResponse {
+  int32 status = 1;
+}
+
+message ResetToDefaultRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+}
+
+message ResetToDefaultResponse {
+  int32 status = 1;
+}
+
+message SelectMeasurementsRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof measurements_enum {
+    MeasurementTypes measurements = 3;
```

Comment:
@reckenro  caught this in #845 . This looks like a bitfield, and there's a way to expose that in `proto` as an array of enums. Not needed for this change.

Reply:
To address this I'll make some changes to grpc-device-scrapigen (starting with https://github.com/ni/grpc-device-scrapigen/pull/105) then copy the resulting metadata to grpc-device in a future PR.